sheet.go

```diff
@@ -497,11 +497,30 @@ func (f *File) SetSheetBackground(sheet, picture string) error {
 		return ErrImgExt
 	}
 	file, _ := os.ReadFile(filepath.Clean(picture))
-	name := f.addMedia(file, ext)
+	return f.setSheetBackground(sheet, ext, file)
+}
+
+// SetSheetBackgroundFromBytes provides a function to set background picture
+// by given worksheet name, extension name and image data
+func (f *File) SetSheetBackgroundFromBytes(sheet, extension string, picture []byte) error {
+	if len(picture) == 0 {
```

Comment:
The error message was not accurate, using the `ErrParameterInvalid` instead of it will be better, and we need add a test for this case.